test/integration_test.js

```diff
@@ -33,9 +33,6 @@ describe("Integration tests", function() {
   this.timeout(0);
 
   before(() => {
-    if (skipLocalServer) {
-      return;
-    }
```

Comment:
Well, that means always instantiating a local test server while `skipLocalServer` denotes the developer wants to use another running local instance. Ideally we should rather add a test for it before calling `server.flush()`, though that would means flushing the user local kinto server, which:

1. is a risky operation as developers don't necessarily want their kinto server instance to be emptied entirely
2. requires calling the flush endpoint manually using a http request

At that point I suspect this is getting too risky and complicated. I'd be in favor of partially reverting #175, namely https://github.com/Kinto/kinto-http.js/pull/175/commits/bd19451059a04f1f5281912fd8db79bb718a967e.

Thoughts?